modules/dynamic-apiserver/pkg/cli/install_operation.go

```diff
@@ -186,14 +188,22 @@ func runInstall(k8sclient kubernetes.Interface) func(command *cobra.Command, arg
 		klParams.Set(installBackupsEnabledParam, backupsEnabled)
 		klParams.Set(installBackupsSnapshotsEnabledParam, snapshotsEnabled)
 
-		var cSite, cKey, kuberlogicDomain string
+		var cSite, cKey, cMappingFile, kuberlogicDomain string
+		cachedChargebeeMappingFile := filepath.Join(cacheDir, "manager/mapping-fields.yaml")
 		if cSite, err = getStringPrompt(command, installChargebeeSiteParam, klParams.GetString(installChargebeeSiteParam), false, nil); err != nil {
 			return errors.Wrapf(err, "error processing %s flag", installChargebeeSiteParam)
 		} else if cSite != "" {
 			cKey, err = getStringPrompt(command, installChargebeeKeyParam, klParams.GetString(installChargebeeKeyParam), true, nil)
 			if err != nil {
 				return errors.Wrapf(err, "error processing %s flag", installChargebeeKeyParam)
 			}
+			cMappingFile, err = getStringPrompt(command, installChargebeeMappingParam, klParams.GetString(installChargebeeMappingParam), true, validateFileAvailable)
```

Comment:
Could we add some validation?

- it's should be a valid yaml config
- should contain list of stc/dst strings

In case of broken file -> chargebee-integration will not work properly.

Reply:
Just in case, we can use jsonschema validation (https://github.com/xeipuuv/gojsonschema)

Reply:
Fixed